template/build_command.template.dart

```diff
@@ -29,17 +47,19 @@ class BuildCommand extends Command {
     // You can change the hashType to sha384 or sha512 if you want
 
     hashScripts(hashType: sha256);
+    enforceCSP(shouldEnforce: env.shouldEnforceCSP);
```

Comment:
As mentioned in the other comment i would expect to call enforceCSP only when i really enforce it. Not when i also disable it.
What do you think about only calling `enforceCSP` when `env.shouldEnforceCSP` is true instead of passing it? 🤔